### ISSUE TITLE
8.1.0 not published on NPM

### ISSUE THREAD

**Issue by loonatic** [fabien0102/openapi-codegen#264]
Hi team, first and foremost, awesome work on this package! :) 

As the title mentions, there seems to be a problem on the workflow on the `@openapi-codegen/cli:build`, and because of that, the package is not being published to NPM. 

Error: 

> [!] Error: Invalid value for option "output.file" - when building multiple chunks, the "output.dir" option must be used, not "output.file". To inline dynamic imports, set the "inlineDynamicImports" option.

@fabien0102 could you please take a look?

Thank you,
loonatic

**Comment by loonatic** [comment]
Pull Request: https://github.com/fabien0102/openapi-codegen/pull/265

**Comment by SferaDev** [comment]
I sent a DM to @fabien0102 this week, let's see if he has bandwidth to fix it

**Comment by fabien0102** [comment]
Yeah, I'm looking at it, if this going smooth, I will take this opportunity to remove a bunch of dependencies on the CLI and simplify a bit ;)
